post/gcmpost.script

```diff
@@ -256,9 +256,9 @@ if( $ARCH == 'Linux'  ) then
                               setenv MASTOR lfe
                               setenv MASDIR /u/$LOGNAME/GEOS5.0/$expid
                               setenv BATCHQ normal
-                              set scpvar = "mcp -a"
-                              set cpvar  = "mcp -a"
-                              set mytar  = "mtar"
+                              set scpvar = "cp -a"
```

Comment:
Should `scpvar` be set to `scp` instead of `cp`?

Reply:
@yvikhlya I think it actually isn't an scp. If you follow down the code, it's actually used in an `ssh` command:
```
                  if( $HOST == 'pleiades' ) then
                     echo "ssh ${MASTOR} $scpvar $locdir/$local ${MASDIR}/$archive"    >> $archfile

```
So it's more "the cp used in ssh".